test/Microsoft.AspNet.Mvc.Core.Test/Filters/FormatFilterTest.cs

```diff
@@ -0,0 +1,253 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Mvc.Core.Filters;
+using Microsoft.AspNet.PipelineCore;
+using Microsoft.AspNet.Routing;
+using Microsoft.Net.Http.Headers;
+using Xunit;
+
+#if ASPNET50
```

Comment:
You shouldn't need this. We aren't running these Core tests in AspnetCore50 as of now.